src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -351,14 +360,17 @@ def test_pf_flow_with_variant(self, capsys):
                     "--flow",
                     temp_dir,
                     "--inputs",
-                    "url=https://www.youtube.com/watch?v=o5ZQyXaAv1g",
+                    "url=https://www.youtube.com/watch?v=o5ZQyXaAv1g&hl=de&persist_hl=1",
                     "answer=Channel",
                     "evidence=Url",
                     "--variant",
                     "${summarize_text_content.variant_1}",
                 )
             outerr = capsys.readouterr()
             assert f"Cannot find the variant invalid_variant for {node_name}." in outerr.out
+            copy_file(
+                Path(temp_dir) / "storage_record.json", Path(FLOWS_DIR) / "web_classification" / "storage_record.json"
```

Comment:
why do we need to do this?

Reply:
Here is a trick. since we use temp folder, the generated record will be generated inside the temp folder.

So that we need a way to get the record back.

By the way I'll delete this line, since we'll use a central file managing these records.